src/modules/classAwesome.js

```diff
@@ -0,0 +1,63 @@
+export default class Awesome {
+  constructor() {
+    this.data = {};
+    this.record = [];
+  }
+
+  addRecord(chore) {
+    this.data = { chore };
+    this.record.push(this.data);
+    return this.data;
+  }
```

Comment:
- [ ] [OPTIONAL] this `this.data` variable is not necessary as it is not used anywhere other than `addRecord` method. I'd strongly suggest you remove it and simply use a local variable.